src/check.ts

```diff
@@ -262,10 +259,7 @@ export class ViolationCheck {
     });
   }
   private generateRuleDocUrl(): string {
-    const serviceName = path.basename(path.dirname(this.rulePath));
-    const ruleName = path.basename(this.rulePath, '.guard');
-    const root = 'https://docs.aws.amazon.com/controltower/latest/userguide';
-    return `${root}/${serviceName}-rules.html#${ruleName}-description`;
+    return 'https://github.com/cdklabs/cdk-validator-cfnguard#bundled-control-tower-rules';
```

Comment:
So we can't point to specific rule documentation anymore?

Reply:
The links weren't correct anyway since the rules were converted to table format.